src/amazonica/aws/dynamodbv2.clj

```diff
@@ -38,7 +38,7 @@
 (defn- scalar-attribute-value? [value]
   (and (map? value)
        (= 1 (count value))
-       (= 1 (count (name-of-first-key value)))))
+       (contains? #{:s :b :BOOL :SS :BS :n :NS :l :m :NULL} (name-of-first-key value))))
```

Comment:
@jmglov Is this correct? `(name-of-first-key value)` should return a string, so it seems like the `contains?` check would never be true. I'm getting the following dynamodb-local error, new in amazonica 0.3.14:

```
com.amazonaws.AmazonServiceException: Invalid attribute value type (Service: AmazonDynamoDBv2; Status Code: 400; Error Code: ValidationException; Request ID: 022030b9-6264-4174-a258-cd02d4ced32f)
```

and I believe this change is related. I can file a new issue but I thought I'd check first.